tests/UI/campaigns/functional/BO/13_shopParameters/03_productSettings/02_productPage/01_displayAvailableQuantities.js

```diff
@@ -74,15 +76,23 @@ describe('Display available quantities on the product page', async () => {
       await expect(result).to.contains(productSettingsPage.successfulUpdateMessage);
     });
 
-    it('should check the product quantity on the product page', async function () {
-      await testContext.addContextItem(this, 'testIdentifier', `checkQuantity${index}`, baseContext);
+    it('should view my shop and go to first product page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `viewMyShop${index}`, baseContext);
 
       page = await productSettingsPage.viewMyShop(page);
 
+      const isHomePage = await homePage.isHomePage(page);
+      await expect(isHomePage, 'Home page was not opened').to.be.true;
+
       await homePage.goToProductPage(page, 1);
+    });
+
+    it('should check the product quantity on the product page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', `checkQuantity${index}`, baseContext);
 
       const quantityIsVisible = await productPage.isQuantityDisplayed(page);
       await expect(quantityIsVisible).to.be.equal(test.args.enable);
+
       page = await homePage.closePage(browserContext, page, 0);
```

Comment:
Can you make `Go back to BO` on another step, please? because if the first iteration fails on that step, the test will not be able to continue.